spec/api-browser-window-spec.js

```diff
@@ -1286,6 +1286,21 @@ describe('BrowserWindow module', () => {
         })
         w.loadFile(path.join(fixtures, 'api', 'preload.html'))
       })
+      it('has access to blink APIs', (done) => {
```

Comment:
Can you move this spec to `chromium-spec.js`, we have been isolating any Web standard api specific testing in there.

Also the original issue was delay loading of some these apis after the script context was created, so it would be good to rewrite this test such that it populates all apis on the window object in the preload as well as in the load web page, compare that there are no differences in the result. Which would basically ensure https://github.com/electron/electron/issues/13787#issuecomment-413211266 spec.

Also, since these are not blink specific apis, might be worth rewording as  `has no difference with features on  window object before and after context creation`

Reply:
> Also the original issue was delay loading of some these apis after the script context was created, so it would be good to rewrite this test such that it populates all apis on the window object in the preload as well as in the load web page

Yup this makes sense

> Can you move this spec to chromium-spec.js

`preload` scripts aren't chromium features though? I think it should go in the preload section where it is now

Reply:
> preload scripts aren't chromium features though? I think it should go in the preload section where it is now

Yeah, I was looking at the features it was testing and decided to go with `chromium-spec`, sounds good to have it here.